test/conftest.py

```diff
@@ -61,6 +61,7 @@ def assert_pdf_equal(actual, expected, tmp_path, generate=False):
             if isinstance(expected, (bytes, bytearray)):
                 pdf_file.write(expected)
             else:
+                expected.set_creation_date(EPOCH)
```

Comment:
Why is this needed?

Reply:
See my other pull request.  Without this, the tests fail due to a bad creation date.